CWE/events/Central Asia.txt

```diff
@@ -60,7 +60,7 @@ country_event = {
 		is_vassal = no
 		war = no
 		
-		NOT = { OR = { ruling_party_ideology = liberal ruling_party_ideology = progressive } }
```

Comment:
Liberal ideology should not be allowed for this event as one of its options is to form a dictatorship.